cypher/cypher-docs/src/docs/dev/ql/administration/security/dbms/role-management-syntax.asciidoc

```diff
@@ -40,3 +40,6 @@ GRANT ROLE MANAGEMENT
 | Enable the specified role or roles to create, delete, assign, remove and list roles
 
 |===
+
+[NOTE]
+Only the `admin` role has access to execute the `SHOW PRIVILEGES` command.
```

Comment:
Maybe `SHOW ROLE xxx PRIVILIGES` as that is what is shown below

Reply:
This is an admonition and applies to the entire section, not just the example directly below. Do you see what I mean?